ui/app/pages/settings/contact-list-tab/add-contact/add-contact.component.js

```diff
@@ -126,6 +127,21 @@ export default class AddContact extends PureComponent {
               </div>
             )}
           </div>
+
+          <div className="address-book__view-contact__group">
+            <div className="address-book__view-contact__group__label">
+              {t('memo')}
+            </div>
+            <TextField
+              type="Multiline"
+              id="memoInput"
+              placeholder="Add memo"
+              rows={4}
```

Comment:
The underlying Material UI Text Field requires the `multiline` prop to be set to `true` or this will be ignored. I don't think `type="Multiline"` is valid :thinking:

Reply:
Whoops 😂 😂 , thanks for the correction!

Reply:
I've fixed this but my text area isn't inheriting the new height established by the row prop. Any suggestions?

Reply:
Updated, changed to a text area but my scss isn't being inherited/applied into the component. Mind taking a look? 😅